flow/Knobs.cpp

```diff
@@ -309,6 +309,8 @@ void FlowKnobs::initialize(Randomize randomize, IsSimulated isSimulated) {
 	if ( randomize && BUGGIFY) { ENCRYPT_CIPHER_KEY_CACHE_TTL = deterministicRandom()->randomInt(2, 10) * 60; }
 	init( ENCRYPT_KEY_REFRESH_INTERVAL,   isSimulated ? 60 : 8 * 60 );
 	if ( randomize && BUGGIFY) { ENCRYPT_KEY_REFRESH_INTERVAL = deterministicRandom()->randomInt(2, 10); }
+	init( ENCRYPT_KEY_HEALTH_CHECK_INTERVAL,                    10 );
+	init( EKP_REQUEST_TIMEOUT,                      		   10.0);
```

Comment:
nit: spacing doesn't seem right?

Also, am not sure if `EKP_REQUEST_TIMEOUT` name is self explanatory; given consumer is HEALTH check maybe we want to name it explicitly, otherwise, to a reader it gives a notion that all EKP calls have that timeout (?)